src/plugins/data/public/search/session/session_indicator/search_session_indicator/components/search_session_name/search_session_name.tsx

```diff
@@ -38,7 +38,7 @@ export const SearchSessionName: React.FC<SearchSessionNameProps> = ({ name, edit
       justifyContent={'spaceBetween'}
       gutterSize={'none'}
       // padding to align with compressed input size
-      style={{ paddingTop: 4, paddingBottom: 4 }}
+      css={{ paddingTop: 4, paddingBottom: 4 }}
```

Comment:
Ideally we'd replace static numbers with EUI tokens (e.g. `euiTheme.size.xs`), but that's probably out of scope for this PR. Just mentioning it here to spread the knowledge/for the future